rust/connlib/tunnel/src/messages/client.rs

```diff
@@ -539,320 +393,52 @@ mod tests {
             "ref": null,
             "topic": "client"
         }"#;
-        let ingress_message: PhoenixMessage<IngressMessages, ReplyMessages> =
-            serde_json::from_str(message).unwrap();
-        assert_eq!(m, ingress_message);
-    }
-
-    #[test]
-    fn messages_ignore_additional_fields() {
```

Comment:
I've deleted all these tests without replacement. Unknown fields in a **struct** are always ignored.